.gitignore

```diff
@@ -1,6 +1,7 @@
 .build_cache/
 build/
 dist/
+lib/*.jl
```

Comment:
Small note here. The Julia build artifacts we care about are in `src/` the ones in lib were copied over by Babel pre-1.13.0 in dev but the equivalent change in master's .gitignore file hadn't been added.